src/interfaces/GeoChart.ts

```diff
@@ -54,6 +54,20 @@ export interface IGeoData {
     tooltipText?: IGeoAttributeItem;
 }
 
+export type IGeoConfigViewportArea =
+    | "auto" // default, Include all data
+    | "continent_af"
+    | "continent_as"
+    | "continent_au"
+    | "continent_eu"
+    | "continent_na"
+    | "continent_sa"
+    | "world";
```

Comment:
Please give comment for these codes like [here](https://github.com/gooddata/gooddata-react-components/pull/1446/files#diff-fa62e9a6886bb2448106a41bbcfa6e89R7)

Reply:
updated.